frontend/components/panels/WalletVatMkrDepositCheckPanel.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <WalletMkrDepositCheckPanel
+            :is-correct.sync="isEnoughMkrInWallet"
+            :wallet-mkr="walletMkr"
+            :wallet-vat-mkr="walletVatMkr"
+            :desired-amount="desiredAmount"
+            :network="network"
+            :token-address-mkr="tokenAddressMkr"
+            :disabled="disabled"
+            :is-loading="isLoading"
+            :is-explanations-shown="isExplanationsShown"
+            @refresh="$emit('refresh')"
+        />
+        <AllowanceAmountCheckPanel
+            :is-correct.sync="isAmountWithinAllowance"
+            :allowance-amount="allowanceAmount"
+            :desired-amount="desiredAmount"
+            currency="MKR"
+            :is-loading="isLoading"
+            :disabled="disabled || !isEnoughMkrInWallet"
+            :is-explanations-shown="isExplanationsShown"
+            @setAllowanceAmount="$emit('setAllowanceAmount', $event)"
+        />
+        <TextBlock v-if="isExplanationsShown && !isEnoughDeposited" class="mt-2">
+            The amount of <format-currency :value="desiredAmount" currency="MKR" /> is not present in the VAT and needs
+            to be deposited there before the participation can happen.
+        </TextBlock>
+        <Button
+            type="primary"
+            class="w-full mt-2"
+            :disabled="disabled || !isAmountWithinAllowance || !isEnoughMkrInWallet || isEnoughDeposited"
+            :is-loading="isLoading"
+            @click="$emit('deposit')"
+        >
+            Deposit <format-currency :value="desiredAmount" currency="MKR" class="ml-1" />
```

Comment:
Same `Minimum to deposit` value should also be displayed here.
As well as emitted as value together with `deposit` event

Reply:
Agree - missed that in my review :( 

Thx for pointing it out!